windows/firefox.admx

```diff
@@ -3896,20 +3917,41 @@
         <decimal value="0"/>
       </disabledValue>
     </policy>
+    <policy name="HandlersOneLine" class="Both" displayName="$(string.HandlersOneLine)" key="Software\Policies\Mozilla\Firefox" explainText="$(string.Handlers_Explain)" presentation="$(presentation.JSONOneLine)">
+      <parentCategory ref="firefox"/>
+      <supportedOn ref="SUPPORTED_FF78"/>
+      <elements>
+        <text id="JSONOneLine" valueName="Handlers" maxLength="16384"/>
```

Comment:
If both policies using the same value name in registry it might break the reports as the GPO report can't safely assign the value to the policy anymore.

Reply:
Unfortunately I don't know of a way to work around that (the reports problem)

And yes, they are combined into one string (with carriage returns)

![image](https://github.com/mozilla/policy-templates/assets/345868/a643bf14-a7dc-45d5-ad39-baab146155f4)
![image](https://github.com/mozilla/policy-templates/assets/345868/ea7b29bf-f616-4447-81fa-16c173760c8f)